---
thr_infrastructure_msgs/RunRobotActionActionResult result